pull_request_template.md

```diff
@@ -17,11 +17,11 @@ The following are ALL required for the PR to be merged:
 Author:
 - [ ] Linked to this PR in JIRA ticket
 - [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
-- [ ] Verified cross-browser compatibility
+- [ ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Internet Explorer
```

Comment:
Wondering which front end dev is going to approve this? :)

Reply:
If we are adding safari why not add edge? is this list base off our GA user base?

Reply:
We can include Edge too - that would complete our top five according to analytics.